build/patch-charts.sh

```diff
@@ -467,6 +467,39 @@ function copyGlobalValues() {
   cp "${SOURCE_DIR}/resources/helm/overlays/global.yaml" "${SOURCE_DIR}/resources/helm/v2.3/"
 }
 
+# This hack is hopefully only needed for a few versions until this PR is merged: https://github.com/istio/istio/pull/37264
```

Comment:
that PR has been closed. maybe link to https://github.com/istio/istio/pull/39375?